articles/cosmos-db/consistency-levels.md

```diff
@@ -1,3 +1,4 @@
+[Az_Consistency Levels.xlsx](https://github.com/MicrosoftDocs/azure-docs/files/10472516/Az_Consistency.Levels.xlsx)
```

Comment:
ms.date : 01/21/23
ms.topic :  summary
author : vishve
title : Summary Consistency Levels
ms.author: v2visve
ms.service 
description: The five consistency levels summarised in balance  with eventual consistency, availability, and latency trade-offs.